include/spdk/blob.h

```diff
@@ -1055,6 +1055,14 @@ void spdk_bs_iter_first(struct spdk_blob_store *bs,
 void spdk_bs_iter_next(struct spdk_blob_store *bs, struct spdk_blob *blob,
 		       spdk_blob_op_with_handle_complete cb_fn, void *cb_arg);
 
+/**
+ * Get the parent blob by using the current blob. 
+ *
+  * \param blob The current blob.
+  * 
+  * \return if parent blob exist send blob else NULL.
```

Comment:
Seems like there's an extra space?

```suggestion
 * \param blob The current blob.
 * 
 * \return if parent blob exist send blob else NULL.
```